project-base/.env

```diff
@@ -14,5 +14,23 @@
 # https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
 
 IGNORE_DEFAULT_ADMIN_PASSWORD_CHECK='0'
-REDIS_PREFIX=''
+DATABASE_HOST=postgres
+DATABASE_PORT=5432
+DATABASE_NAME=shopsys
+DATABASE_USER=root
+DATABASE_PASSWORD=root
+ELASTICSEARCH_HOST='elasticsearch:9200'
 ELASTIC_SEARCH_INDEX_PREFIX=''
+REDIS_HOST=redis
+REDIS_PREFIX=''
+
+MAILER_TRANSPORT=smtp
+MAILER_HOST=smtp-server
+MAILER_USER=
+MAILER_PASSWORD=
+MAILER_DELIVERY_WHITELIST=/@shopsys\.com$/
+MAILER_DISABLE_DELIVERY=0
```

Comment:
I can be tempted to set `false` here and I think it won't work. Not sure whether it is a problem

Reply:
I am not really sure if there is something what we can do about it. The ENVs are string only.

Reply:
Symfony uses env processor (:bool) which accepts `no`, `false`, etc., so that's why I can be tempted to do this here.

But I'm not sure if it's something we can do here. Maybe a comment or documentation

Reply:
As this is problem only when ENV is used via `getenv()` and this is only usage I even more hacked our hack.

IMHO it is ok to do that without any regrets as it is for unified usages of ENVs and to prevent confusion or misconfiguration.

So what is it?

Simply retype the returned value from `getenv()` and check if it is not `string(5) "false"`.

```
$envMailerDisableDelivery = getenv('MAILER_DISABLE_DELIVERY');
        if ($envMailerDisableDelivery !== false) {
            $castedEnvMailerDisableDelivery = (bool)$envMailerDisableDelivery === true && $envMailerDisableDelivery !== 'false';
            $container->setParameter('mailer_disable_delivery', $castedEnvMailerDisableDelivery);
        }
```

Are you OK with that? It will do much better than some tiny note in doc. And even better to see happy developers.

Reply:
Implemented in 71e13a9cdec3d6e81dccd8332766181005ee1f1a

Reply:
That's cool. I did take a look at the implementation of the env processor in Symfony (`\Symfony\Component\DependencyInjection\EnvVarProcessor`) and for bool they're using 
```
(bool) (filter_var($env, \FILTER_VALIDATE_BOOLEAN) ?: filter_var($env, \FILTER_VALIDATE_INT) ?: filter_var($env, \FILTER_VALIDATE_FLOAT));
```
does it make sense to use this for `$castedEnvMailerDisableDelivery`? It will be unified with the way, how Symfony handles boolean environment variable values (for example 'no' is also false)

Reply:
Nice! Fixed in 4ade8e8da9fae85d4242ed6c764f4ff33f9724b3